drupal/rootfs/etc/islandora/utilities.sh

```diff
@@ -246,6 +246,23 @@ function install_site {
         chmod a=rwx "${site_directory}/settings.php"
     fi
 
+    echo "--driver ${driver}"
+    echo "--host ${host}"
+    echo "--port ${port}"
+    echo "--dbuser ${user}"
+    echo "--dbpassword ${password}"
```

Comment:
We probably shouldn't print passwords to std-out

Reply:
Any suggestions? (I'm at a loss, I don't have enough Docker management experience to know what the risk is or how to do it better.)